docs/src/routes/docs/components/file-dropzone.svx

```diff
@@ -76,6 +76,11 @@ All `<FileDropzone>` components use the `.file-dropzone` class.
 
 The inner content of the dropzone when no files are selected. Defaults to a message that responds to the state of the dropzone: empty, with files ready to be dropped, with incorrect files ready to be dropped or disabled.
 
+| Prop Name | Description |
```

Comment:
How about adding `Type` column instead of having to state "The boolean state ..." in the description?

Reply:
Reasonable